ros_cross_compile/docker/build_workspace.sh

```diff
@@ -9,6 +9,7 @@ trap 'cleanup' EXIT
 
 mkdir -p /opt/ros/"${ROS_DISTRO}"
 touch /opt/ros/"${ROS_DISTRO}"/setup.bash
+mv /root/defaults.yaml .
```

Comment:
Based on my previous comment, this line could then be removed

Reply:
Yep, done. Thanks!